### PR TITLE
Add search embeddings extraction as part of the age_estimation workflow

### PR DESCRIPTION
**Pull request by munkyshi**
### Linked issue(s):
Fixes KOL-3500

### What change does this PR introduce and why?
Introduce a documentation example for extracting search embeddings on image data.

### Please check if the PR fulfills these requirements

- [x] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [x] Relevant docs have been added / updated

### CODE REVIEW

**Comment by munkyshi** [comment]
Hmmm... I can't merge this since everyone else part of `kolenaIO/kolena-owners` is OOTO 😅

**Comment by munkyshi** [comment]
Please message me on Slack if you want a `kolena_studio_client_api.tar.gz` tarball for testing

**Comment by y27choi** [comment]
Should we mention this example somewhere from our docs? I wonder if there should be a page under advanced usage section for this.

File: examples/search_embeddings/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+> Kolena supports search embedding extraction and upload as an opt-in feature for our customers.
+> Please message your point of contact for the latest relevant extractor package.
+
+This example runs against data from the [`age_estimation`](../age_estimation) example workflow, and assumes this
+data has already been uploaded to your platform.
+
+## Setup
+
+1. Ensure that data for the [`age_estimation`](../age_estimation) workflow has been seeded.
+2. Copy the `kolena_studio_client_api.tar.gz` file (provided by your Kolena contact) to the `./local_packages` directory.
```
Comment (y27choi) [inline examples/search_embeddings/README.md]: I know @gordonhart wanted to rename this file. Is now a good time to change the name of this package?

Comment (y27choi) [inline examples/search_embeddings/README.md]: We need to make sure the copy that we provide is named exactly `kolena_studio_client_api.tar.gz`. `kolena_studio_client_api-0.3.2.tar.gz` didn't work when `poetry install`.

File: examples/search_embeddings/search/extract_embeddings.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    local_path: str = args.local_path
+
+    if local_path is None or local_path == "":
+        print(
+            "local-path argument is unset. Please note that pre-downloading all images in batch and later "
+            "extracting embeddings with the local-path flag will be significantly faster than streaming the "
+            "extraction.",
+        )
+
+    s3_path = f"s3://{BUCKET}/{DATASET}/imgs/"
```
Comment (y27choi) [inline examples/search_embeddings/search/extract_embeddings.py]: `s3_path` is defined twice.

File: examples/search_embeddings/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+> Kolena supports search embedding extraction and upload as an opt-in feature for our customers.
+> Please message your point of contact for the latest relevant extractor package.
+
+This example runs against data from the [`age_estimation`](../age_estimation) example workflow, and assumes this
+data has already been uploaded to your platform.
+
+## Setup
+
+1. Ensure that data for the [`age_estimation`](../age_estimation) workflow has been seeded.
```
Comment (y27choi) [inline examples/search_embeddings/README.md]: Should we make it more explicit that `seed_test_suite.py` has to be called first?

**Review by y27choi — APPROVED** [review]
Left some minor comments but the code looks good. Also tested out locally following the instruction and was able to get the search working on age_estimation dataset.

**Comment by munkyshi** [comment]
> Should we mention this example somewhere from our docs? I wonder if there should be a page under advanced usage section for this.

Planning on that as the next step!

File: examples/search_embeddings/README.md (hunk 1 above)
Comment (munkyshi) [reply examples/search_embeddings/README.md]: Let's do it -- do you know what name he wanted? Otherwise I can throw something general together.

Comment (y27choi) [reply examples/search_embeddings/README.md]: I don't think he had a name in mind. I'm sure you can come up with a better name than what we currently have.

File: examples/search_embeddings/README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+# Example Extraction: Image Search Embeddings
+
+> Kolena supports search embedding extraction and upload as an opt-in feature for our customers.
+> Please message your point of contact for the latest relevant extractor package.
+
+This example runs against data from the [`age_estimation`](../age_estimation) example workflow, and assumes this
+data has already been uploaded to your platform.
```
Comment (marko-polo-cheno) [inline examples/search_embeddings/README.md]: Only additional comment is that "your platform" could be vague. I'd suggest `your Kolena platform`.

**Review by marko-polo-cheno — APPROVED** [review]
LGTM, did not test

File: examples/search_embeddings/README.md (hunk 4 above)
Comment (munkyshi) [reply examples/search_embeddings/README.md]: Done.

File: examples/search_embeddings/README.md (hunk 3 above)
Comment (munkyshi) [reply examples/search_embeddings/README.md]: Done.

File: examples/search_embeddings/README.md (hunk 1 above)
Comment (munkyshi) [reply examples/search_embeddings/README.md]: Renamed to `kolena-embeddings.tar.gz`

File: examples/search_embeddings/search/extract_embeddings.py (hunk 2 above)
Comment (munkyshi) [reply examples/search_embeddings/search/extract_embeddings.py]: Done.

**Pull request merged**